core/src/cnf/mod.rs

```diff
@@ -42,7 +42,6 @@ pub static INSECURE_FORWARD_RECORD_ACCESS_ERRORS: Lazy<bool> =
 #[cfg(any(
 	feature = "kv-mem",
 	feature = "kv-surrealkv",
-	feature = "kv-file",
```

Comment:
It isn't really, but this feature no longer exists and I run the newest version of rust which generates warnings for non-existent features so test wouldn't run because it would error because of `#[deny(warnings)]`.

Reply:
I could remove these changes, but since their just a relatively small fix for warnings I left it in.